backends/platform/android/org/scummvm/scummvm/ScummVMActivity.java

```diff
@@ -1063,6 +1075,64 @@ public void onStart() {
 		super.onStart();
 	}
 
+	@Override
+	protected void onNewIntent(Intent intent) {
+//		Log.d(ScummVM.LOG_TAG, "onNewIntent: " + intent.getData());
+
+		super.onNewIntent(intent);
+
+		Uri intentData = intent.getData();
+
+		// No specific game, we just continue
+		if (intentData == null) {
+			return;
+		}
+
+		// Same game requested, we continue too
+		if (intentData.equals(getIntent().getData())) {
+			return;
+		}
+
+		setIntent(intent);
+
+		if (_events == null) {
+			finish();
+			startActivity(intent);
+			return;
+		}
+
+		// Don't finish our activity on C++ end
+		_finishing = true;
+
+		_events.clearEventHandler();
+		_events.sendQuitEvent();
+
+		// Make sure the thread is actively polling for events
+		_scummvm.setPause(false);
+		try {
+			// 1s timeout
+			_scummvm_thread.join(2000);
```

Comment:
A small issue, but there is discrepancy between the comment (1s timeout) and the actual timeout in join (2000ms, ie. 2 seconcds)

Reply:
Thanks! Fixed in both places (as this piece of code was a copy-paste)